test/unit/active_model/serilizable_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+module ActiveModel
+  class SerializableTest
+    class InstrumentationTest < Minitest::Test
+      def setup
+        @events = []
+
+        ActiveSupport::Notifications.subscribe("!serialize.active_model_serializers") do |name, start, finish, id, payload|
+          @events << { name: name, serializer: payload[:serializer] }
+        end
+      end
+
+      def teardown
+        ActiveSupport::Notifications.unsubscribe("!serialize.active_model_serializers")
+      end
+
+      def test_default_serializer
+        DefaultSerializer.new(1).as_json
+
+        assert_equal @events, [{ name: '!serialize.active_model_serializers', serializer: 'ActiveModel::DefaultSerializer' }]
```

Comment:
expectation in assert_equal should be on the left http://docs.seattlerb.org/minitest/Minitest/Assertions.html